app/src/ui-blokada/res/values/strings_notification.xml

```diff
@@ -18,7 +18,7 @@
     <!-- Parameter: branded app name (Blokada). -->
 
     <string name="notification_keepalive_text">Keep alive</string>
-    <string name="notification_keepalive_description">Activate this switch if you experience problems with app staying active. The app will enable a permanent notification to prevent the system from killing it.</string>
+    <string name="notification_keepalive_description">Activate this switch if you experience problems with the app staying active in the background. Blokada will enable a permanent notification to prevent the system from killing it\; you can hide it by choosing Notifications above and there Settings.</string>
```

Comment:
no need for a backslash

Reply:
Fixed with 41a53ae20add8b5b199bdfd52b9a8b85edf2e5b5.